### ISSUE TITLE
Create GH Action to build & update the Docker image

### ISSUE THREAD

**Issue by michaelosthege** [pymc-devs/pymc#5939]
In #5881 the `Dockerfile` was updated to work for PyMC v4+, and @danhphan also pushed them to https://hub.docker.com/r/pymc/pymc/tags manually.

We should add a GitHub Actions pipeline to automatically build a new image on new releases, and push it tagged as `latest` and with the version.

**Comment by twiecki** [comment]
@danhphan Is this something you can help with?

**Comment by danhphan** [comment]
Hi @twiecki I would love to do it, but current really busy as I have two papers' submission deadline at the end of this month. And also need to focus on the GSoC project.

Could be great if someone can go ahead to do it. Otherwise, I will do it early next month. Thanks all.

**Comment by symeneses** [comment]
I would like to work on this 🙌🏽

**Comment by 5hv5hvnk** [comment]
@symeneses  I didn't realise you were working on it too
I wrote some of the script haven't tested it yet (but I am hopeful it will work fine) you can go through it once
```
name: Push Dockerfile to docker docker hub
on:
	push:
		paths:
			- 'scripts/Dockerfile'

jobs:
	build-container:
		name: Build docker image
		runs-on: ubuntu-latest
		steps:
			- name: Checkout code
			  uses: actions/checkout@v2
			- name: Publish tp docker hub
			  uses: docker/build-push-actions@v1
			  with:
			  	username: ${{ secrets.Docker-Username}}
			  	password: ${{ secrets.Docker-Password}}
			  	repository: pymc/pymc
			  	tags: latest
```

We will have to add secrets for docker password and username rest seems fine to me

**Comment by symeneses** [comment]
@5hv5hvnk all good, I just opened a PR for this #5966.

**Comment by michaelosthege** [comment]
Assuming that the fixes from #5977 will succeed, I'm closing this 🎉
